City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-9459: new option for school search filter

# [UHF-9459](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9459)


## What was done


* This thing was fixed

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-9459-school-search`
  * `composer require drupal/hdbt:dev-UHF-9459-school-search`
  * `make fresh`
* Run `make drush-cr`

## How to test


* [x] Run: `drush migrate:import tpr_ontology_word_details --reset-threshold 1`
* [x] Run: `drush migrate:import tpr_unit --reset-threshold 1 --idlist=6667:en,6667:sv,6667:fi
`
* [x] Go to https://helfi-kasko.docker.so/fi/kasvatus-ja-koulutus/admin/content/integrations/tpr-unit/6667/edit.
  * You should see new Ontologyword details items.
* [x] Check that https://helfi-kasko.docker.so/fi/kasvatus-ja-koulutus/ressun-peruskoulu looks ok.
  * Make sure that NEW ontology word details are not printed to the page.
* Index schools to elastic search 
* Go to school search: https://helfi-kasko.docker.so/fi/kasvatus-ja-koulutus/perusopetus/peruskoulut
  * [x] Test search with school information > Bilingual education filter.  
  * [x] Check translations work. 

## Other PRs


* https://github.com/City-of-Helsinki/drupal-hdbt/pull/932


[UHF-9459]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9459?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
